File: coin.py
import random

    # 앞면과 뒷면의 횟수를 저장할 변수를 초기화합니다.
front = 0
back = 0

for _ in range(100):
    # 0 또는 1을 무작위로 생성
    coin = random.randint(0, 1)

    if coin == 0:
        front += 1
    else:
        back += 1

print("동전의 앞면은 %d번 나왔습니다."% front)
print("동전의 뒷면은 %d번 나왔습니다."% back)

File: coin.c
#include <stdio.h>
#include <stdlib.h> // random 함수를 쓰기위한 헤더파일
#include <time.h> // 초기화 함수를 위한 헤더파일

int main() {
    srand(time(0)); // 난수 생성 초기화를 별도로 생성, ;없이 작성

    int front = 0; // Python은 별도로 변수 생성 X
    int back = 0;

for (int i = 0; i < 100; i++) { // Python과 for문 문법이 다름, {}를 안쓰고 들여쓰기를 이용하여 구분
        int coin = rand() % 2; // Python과 다르게 randon 함수에서 %를 이용하여 0, 1을 선별

        if (coin == 0) { //Python과 다르게 ()를 안쓰고 작성
            front++; // Python은 증감 연산자 X
        } else {
            back++;
        }
    }

    printf("동전의 앞면은 %d번 뒷면은 %d번", front, back); // Python은 print로 작성, 변수 출력시 format를 이용하여 변수 출력

    return 0;
}